specification/synapse/data-plane/readme.md

```diff
@@ -75,6 +75,7 @@ input-file:
 - Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/SparkJobDefinition.json
 - Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/SqlScript.json
 - Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Trigger.json
+- Microsoft.Synapse/preview/2019-06-01-preview/gitintegration.json
```

Comment:
are these and other changes in data-plane folder intentional? if so, please include them in a separate PR as that does not need ARM review

Reply:
All these changes are related need to checkin together, from ARM prospective please review it.